### PR TITLE
Updated nexus standards, shacl and prov-o contexts

### PR DESCRIPTION
**Pull request by MFSY**

### CODE REVIEW

**Comment by MFSY** [comment]
The shacl and standard contexts will be obtained from nexus-common once [this](https://github.com/BlueBrain/nexus-commons/pull/36) pull request is merged

File: modules/prov/src/main/resources/contexts/nexus/core/provo20130430/v0.1.0.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      },
+      "hadMember": {
+        "@id": "prov:hadMember"
+      },
+      "wasGeneratedBy": {
+        "@id": "prov:wasGeneratedBy"
+      }
+    }
+  ]
+}
```
Comment (annakristinkaufmann) [inline modules/prov/src/main/resources/contexts/nexus/core/provo20130430/v0.1.0.json:25]: For the ephys provenance template, we also have a prov:activity relationship between StimulusExperiment and TraceGeneration

**Review by annakristinkaufmann — APPROVED** [review]
Looks good to me!

File: modules/prov/src/main/resources/contexts/nexus/core/provo20130430/v0.1.0.json (hunk 1 above)
Comment (MFSY) [reply modules/prov/src/main/resources/contexts/nexus/core/provo20130430/v0.1.0.json:25]: True. We will update the context later.

**Pull request merged**